portfolio/src/main/webapp/script.js

```diff
@@ -26,21 +26,21 @@ function getRandomQuote() {
     // see function doc on line 57
     revertToUnchecked();
 
-    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    let quote = quotes[Math.floor(Math.random() * quotes.length)];
   
-    let entrance = entrances[Math.floor(Math.random() * greetings.length)];
+    let entrance = entrances[Math.floor(Math.random() * entrances.length)];
 
```

Comment:
Looks like there are many empty lines, which are unnecessary. Try to group the code into blocks.

Reply:
blocked some code together, I still like to have some empty lines between different blocks to contrast the functionalities